inc/autoload.function.php

```diff
@@ -329,6 +329,15 @@ function glpi_autoload($classname) {
 
    $dir = GLPI_ROOT . "/inc/";
 
+   //hack for \Event
+   //@since 9.3.1 -- WILL BE REMOVED IN FUTURE RELEASE
+   if ($classname === 'Event') {
```

Comment:
Not sure this is OK if the autoload call is made from a file that is in the Glpi namespace. Anyways; since GLPI does not uses namespaces at all, I guess this has will removed before :)

Reply:
Exactly: https://github.com/glpi-project/glpi/pull/4597